mantle/cmd/kola/testiso.go

```diff
@@ -226,19 +226,10 @@ func runTestIso(cmd *cobra.Command, args []string) error {
 		targetScenarios[scenario] = true
 	}
 
-	// ppc64le: pxe-install does not work: https://github.com/coreos/coreos-assembler/issues/1457. Seems like
-	// the SLOF doesn't like the live initramfs image.
-	// s390x: pxe-install does not work because the bootimage used today is built from the rhcos kernel+initrd
-	// since s390x does not have a pre-built all in one image. For the pxe case, this image turns out to
-	// be bigger than the allowed tftp buffer. iso-install does not work because s390x uses an El Torito image
-	switch system.RpmArch() {
-	case "s390x":
-		fmt.Println("Skipping pxe-install and iso-install on s390x")
+	// s390x: iso-install does not work because s390x uses an El Torito image
```

Comment:
Is this still true?  The s390x ISO image does include an El Torito image.

Reply:
yes. the coreos-installer doesn't recognize it as a valid iso and bails out when trying to embed the ignition config because for s390x, an el torito image is used with its own initramfs: https://github.com/coreos/coreos-assembler/blob/master/src/cmd-buildextend-installer#L526. Is there some other way we can inject the ignition config ?

Reply:
Ah, right, Ignition injection.  No, there's no other mechanism for the ISO.  We should probably see if we can make the custom ISO header point to the padding area _within the El Torito image_ on s390x.